ios/engine/KMEI/KeymanEngineTests/KMPJSONTests.swift

```diff
@@ -127,4 +129,157 @@ class KMPJSONTests: XCTestCase {
     let nrc_en_mtnt: KMPMetadata = try loadObjectFromJSON(at: TestUtils.PackageJSON.kmp_json_nrc_en_mtnt)
     kmp_info_nrc_en_mtnt_assertions(nrc_en_mtnt)
   }
+
+  func testKMPKeyboardFrom() throws {
+    guard let constructedMetadata = KMPKeyboard(from: TestUtils.Keyboards.khmer_angkor) else {
+      XCTFail("Could not construct KMPKeyboard from InstallableKeyboard")
+      return
+    }
+
+    if let _ = KMPKeyboard(from: TestUtils.LexicalModels.mtnt) {
+      XCTFail("Constructed KMPKeyboard from InstallableLexicalModel")
+    }
+
+    // The original version extracted from a KMP.
+    let kmpMetadata: KMPKeyboard = try loadObjectFromJSON(at: TestUtils.PackageJSON.keyboard_khmer_angkor)!
+
+    XCTAssertEqual(constructedMetadata.id, kmpMetadata.id)
+    XCTAssertEqual(constructedMetadata.name, kmpMetadata.name)
+    XCTAssertEqual(constructedMetadata.version, kmpMetadata.version)
+    // The pre-extracted version currently does not have fonts set, so the following two lines will fail.
+//    XCTAssertEqual(constructedMetadata.font, kmpMetadata.font)
```

Comment:
The newer version of khmer_angkor uses Mondulkiri
https://api.keyman.com/keyboard/khmer_angkor

Reply:
Yep.  It's already used for some of the tests.  I find having the two different definitions handy for some unit testing aspects.